server/app/views/applicant/ApplicantUpsellTemplate.html

```diff
@@ -12,9 +12,15 @@ <h1 th:text="#{title.applicationConfirmation}"></h1>
       ></p>
       <a
         href="/"
-        class="usa-button"
+        class="usa-button margin-y-2"
         id="another-program"
         th:text="#{link.applyToAnotherProgram}"
+      ></a>
+      <a
+        th:href="${downloadHref}"
+        class="usa-button margin-y-2"
+        id="download-pdf"
+        th:text="#{button.downloadPdf}"
```

Comment:
Just a qq - are we planning on adding translations to the text? I understand this is still in works.

Reply:
Yes, `#{button.downloadPdf}` means that the text will be translated